docs/toc.md

```diff
@@ -0,0 +1,19 @@
+- <h3>[Home]({{urlRoot}}/index)</h3>
+- <h3>Get started</h3>
+    - [Installation and setup]({{urlRoot}}/setup-and-installing)
+    - [Feature list]({{urlRoot}}/features)
+- <h3>Concepts</h3>
+    - [Interop Code Generator]({{urlRoot}}/content/interop)
+    - [Actor Property Handover]({{urlRoot}}/content/handover-between-server-workers)
+    - [Singleton Actor]({{urlRoot}}/content/singleton-actors)
+- <h3>[Porting to the GDK]({{urlRoot}}/content/porting-unreal-project-to-gdk)</h3>
```

Comment:
I think this looks strange on its own - I didn't appreciate quite how strange it would look! Maybe move it into "Get started", between the two existing pages?

Reply:
And maybe change the name to "Port an Unreal project to the GDK"